### DISCUSSION TITLE
cron scheduled check down after success

### DISCUSSION THREAD

**Discussion by rjrbytes** [healthchecks/healthchecks#730] [Q&A] [answered]
I really love this tool, but have a bit of a puzzle. I have a script that runs and checks in Monday-Friday at 2:45am. The expression I'm using is below and the preview for anticipated check-in times looks correct:
45 2 * * 1-5
I've got the grace period set to an hour, and for some reason, the check consistently goes down at 3:45 even after a successful check-in and I can't figure out why. None of my other (20+) scripts running on the same system appear to have this issue and I've re-created the check from scratch at least twice. I'm running 2.4.1 in Docker. Any ideas on how to troubleshoot this?

-Robert

**Comment by niklas2810** [comment, marked as answer]
Yeah, you can take a look at the log at the right. Some ideas from my side:

- Double-check your cron config on the healthchecks site. Also, make sure to set it to the correct timezone if your server is not on UTC!
-  Verify when a ping arrives. If it arrives at 2:44:59, it doesnt count. Although this most likely won't be the case if you're running a cron job.
- If you're using the `/start` endpoint, make sure the cron job send a finish ping as well.

If none of this works, could you please show what you mean by "successful check-in"? Maybe with a screenshot, make sure to redact sensitive information though.

**Reply by rjrbytes** [reply]
I'm pretty sure it's the second item. My pings have been arriving at 2:44:xx due to a small difference in clocks between the calling system and the healthchecks server. I hadn't realized that early pings didn't count.

Thanks!

**Reply by niklas2810** [reply]
@rjrbytes Okay, great! I know I'm a bit late to return to this discusison, but you can use NTP to synchronize your server time on a regular basis, which will prevent these events from happening. An alternative would be to modify the healthcheck cron config to await a ping at 2:44 with a bit of grace time.
